### PR TITLE
pkt2flow 1.2

### PR DESCRIPTION
**Pull request by larseggert**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### Changes to Homebrew's Core:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031) if you'd like one.
- [ ] Have you successfully ran `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Formula/pkt2flow.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Pkt2flow < Formula
+  desc "Classify packets into flows"
+  homepage "https://github.com/caesar0301/pkt2flow"
+  url "https://github.com/caesar0301/pkt2flow/archive/v1.2.tar.gz"
+  sha256 "deb4c5dc2fd11c45e338e9d0c0127df03257040127c49016e40aaf491e5674a6"
+
+  depends_on "scons" => :build
+
+  def install
+    "scons"
```
Comment (dunn) [inline Library/Formula/pkt2flow.rb]: This should be unquoted (we have a custom method for running scons).

**Comment by dunn** [comment]
This is failing our notability audit:
```
==> audit problems
pkt2flow:
 * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```
We only waive that requirement for established projects that move to GitHub or in other exceptional circumstances.

**Comment by DomT4** [comment]
Passing on this until it meets our eligibility guidelines I'm afraid. If you wish to support it yourself up to that point it's very easy to support formulae in your own tap and there are some tips on doing that [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) if interested.

Apologies we can't accept this for now, but thank you for the submission to Homebrew nonetheless.

**Comment by larseggert** [comment]
In case anyone is looking, I put this into my own tap for now: https://github.com/larseggert/homebrew-mytap

**Pull request closed without merging**